src/SLCore.Listeners.UnitTests/Implementation/AnalysisListenerTests.cs

```diff
@@ -171,10 +175,10 @@ public void RaiseIssues_HasIssues_PublishIssues(bool isIntermediatePublication)
         }
     }
 
-    private static IRaiseIssueParamsToAnalysisIssueConverter CreateConverter(RaiseIssuesParams raiseIssuesParams, IAnalysisIssue[] issues)
+    private static IRaiseIssueParamsToAnalysisIssueConverter CreateConverter(FileUri fileUri, IAnalysisIssue[] issues)
     {
         var raiseIssueParamsToAnalysisIssueConverter = Substitute.For<IRaiseIssueParamsToAnalysisIssueConverter>();
-        raiseIssueParamsToAnalysisIssueConverter.GetAnalysisIssues(raiseIssuesParams).Returns(issues);
+        raiseIssueParamsToAnalysisIssueConverter.GetAnalysisIssues(fileUri, Arg.Any<IEnumerable<RaisedIssueDto>>()).Returns(issues);
```

Comment:
for second parameter getting the exact argument was not feasible so used arg.any. We check if it's called with correct parameter.